# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/angle/cherry_picks.txt
# Commit: 040a6d3c401d
# Full Hash: 040a6d3c401de4142568a0c942f169244dc72a71
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-12 09:34:21
# Description:
#   Bug 1636122 - [angle] Use deviceContext instead of deviceContext1. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74770
# ==============================================================================

diff -r 9923799dd260 -r 040a6d3c401d gfx/angle/cherry_picks.txt
--- a/gfx/angle/cherry_picks.txt	Tue May 12 01:13:17 2020 +0000
+++ b/gfx/angle/cherry_picks.txt	Tue May 12 01:22:38 2020 +0000
@@ -1,3 +1,12 @@
+commit eabf2a79aac36223b5ab5c5b0810d5522095f0bc
+Author: Jeff Muizelaar <jrmuizel@gmail.com>
+Date:   Mon May 11 13:01:46 2020 -0400
+
+    Use deviceContext instead of deviceContext1 (#24)
+    
+    Sometimes we don't have a deviceContext1 (on Win7) and would crash. We
+    on't have any need to use it so just use deviceContext.
+
 commit 19f1bef071c32df60d9d62a7907c42270d0a3dd7
 Author: Jeff Muizelaar <jrmuizel@gmail.com>
 Date:   Fri May 8 15:08:24 2020 -0400
